object-manipulation-2/src/invert.js

```diff
@@ -0,0 +1,19 @@
+/* exported invert */
+function invert(source) {
+  const newObject = {};
+  if (source === undefined) {
+    return {};
```

Comment:
this guard shouldn't be necessary. if there are no keys in the `source`, then your loop will be skipped and `newObject` will be returned.